core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -765,6 +776,33 @@ private ClientFactoryOptions buildOptions() {
         }
     }
 
+    private static Map<ChannelOption<?>, Object> getDefaultNettyChannelOptions(
+            ClientFactoryOptions clientFactoryOptions, long maxConnectionAgeMillis,
+            long idleTimeoutMillis, long pingIntervalMillis) {
```

Comment:
- How about renaming to `setDefaultChannelOptionsIfAbsent`? IMO, `getDefaultNettyChannelOptions` sounds to return Netty`s the default channel option. 
- nit: Take `Map<ChannelOption<?>, Object>` and return `Map<ChannelOption<?>, Object>` for the clear type signature?
```suggestion
    private static Map<ChannelOption<?>, Object> setDefaultChannelOptionsIfAbsent(
            Map<ChannelOption<?>, Object> channelOptions, long maxConnectionAgeMillis,
            long idleTimeoutMillis, long pingIntervalMillis) {
```